### ISSUE TITLE
Feature ideas 

### ISSUE THREAD

**Issue by Jimmi08** [arunshekher/mentions#8]
I can live without this, your work is awesome, this is just...  just ideas

-  setting in prefs if include chatbox message in email body 

- if sender/writter has public email (not hidden), include it as reply to in email. So if I get notification that somebody mentioned my nick (with chatbox content) I can reply to them directly via email.

**Comment by Jimmi08** [comment]
Feedback from user:  direct link to login in site :) in sended email.  They are lazy :) And big thanks for something like this.

**Comment by arunshekher** [comment]
Link to mentioned content is a [work in progress](https://github.com/arunshekher/mentions/tree/feature-branch/content-links-in-notification), but adding actual message or part of the message in email is not planned (rather it should be said that I dropped that plan) though it cold be easily done. My idea is to draw more user visits to the website directly from mention email keeping them in suspense off the content.

**Comment by Jimmi08** [comment]
Yes, the reason why I mentioned preferencies for this. But I have site where I dont need to draw user to site, they are already there

**Comment by arunshekher** [comment]
The second part of your enhancement idea is really interesting, will look into that if it will not hinder my prime goal with notification email mentioned above or if enough demand from the community. Once again thank you very much with all the valuable feedbacks you are providing. Greatly appreciate it.
